src/types/index.ts

```diff
@@ -100,6 +100,7 @@ export interface Proposal {
   againstVotesWei: BigNumber;
   createdDate: Date | undefined;
   description: DescriptionV1 | DescriptionV2;
+  proposalType: 'Normal' | 'FastTrack' | 'Critical';
```

Comment:
Instead of redeclaring this type in multiple places, you can use an enum:

```ts
enum ProposalTypeName {
    Normal,
    FastTrack,
    Critical
}
```

There currently is a type called `proposalTypeName` at `src/types/index.tsx`, this enum would replace it.